mail.php

```diff
@@ -77,9 +77,10 @@ class Mail{
     * The body can be left blank.
     **/
     public function __construct($to, $from, $subject, $text_content = "", $html_content = ""){
+      date_default_timezone_set('America/Los_Angeles');
```

Comment:
? i think you should know more about utf-8.....every country not use
english as mother language will like it,and your program has bug,so i hope
you can test it.

2016-09-08 14:09 GMT+08:00 Nico <notifications@github.com>:

> In mail.php <https://github.com/stlewis/Mail/pull/6#discussion_r77950586>:
>
> > @@ -77,9 +77,10 @@ class Mail{
> >      * The body can be left blank.
> >      **/
> >      public function __construct($to, $from, $subject, $text_content = "", $html_content = ""){
> > +      date_default_timezone_set('America/Los_Angeles');
>
> Serious? This is not useful for other countries or states
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/stlewis/Mail/pull/6/files/7815e976e268b8d002ee76c3781d93b88f561ccc#r77950586>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARfx4TI2QjGmBwbG_Ucl2VDOE9aH73Gbks5qn6aTgaJpZM4JInBN>
> .
>